mypy/messages.py

```diff
@@ -897,6 +897,10 @@ def typeddict_item_name_not_found(self,
     def type_arguments_not_allowed(self, context: Context) -> None:
         self.fail('Parameterized generics cannot be used with class or instance checks', context)
 
+    def disallowed_any_type(self, typ: Type, context: Context) -> None:
+        self.fail('Expressions of type "Any" are disallowed '
```

Comment:
Updated the error message to say `Expressions of type that contains type "Any" are disallowed (has type {})` if type is not `Any`.